Remotion/Security/Core/Metadata/LocalizationFileNameStrategy.cs

```diff
@@ -17,15 +17,18 @@
 using System;
 using System.Globalization;
 using System.IO;
+using Remotion.Utilities;
 
 namespace Remotion.Security.Metadata
 {
   public class LocalizationFileNameStrategy
   {
-    // TODO RM-7869: Add missing argument checks
 
     public string GetLocalizationFileName (string metadataFilename, CultureInfo culture)
     {
+      ArgumentUtility.CheckNotNull ("metadataFilename", metadataFilename);
```

Comment:
Imo this could be `CheckNotNullOrEmpty`, same for the other method where the check was added. @MichaelKetting